shotover-proxy/tests/valkey_int_tests/basic_driver_tests.rs

```diff
@@ -155,8 +155,8 @@ async fn test_save(connection: &mut Connection) {
         "expected lastsave2 > 0, but lastsave was {lastsave2}"
     );
     assert!(
-        lastsave2 > lastsave1,
-        "expected lastsave2 > lastsave1, but was {lastsave2} > {lastsave1}"
+        lastsave2 >= lastsave1,
+        "expected lastsave2 >= lastsave1, but was {lastsave2} > {lastsave1}"
```

Comment:
does `{lastsave2} > {lastsave1}` need to update as well?

Reply:
nope, because it was originally wrong :laughing: